azure-pipelines.yml

```diff
@@ -71,16 +68,13 @@ stages:
       pool: {vmImage: 'macOS-10.15'}
       variables:
         python.version: '3.7'
-        CIBW_BEFORE_BUILD: pip install -U Cython
```

Comment:
We are losing this, but it shouldn't be needed with the pyproject.toml existing now (it was a holdover from before we had one and before pip 19).

Reply:
Yeah, I thought it wasn't necessary because it's in `build-system.requires`